hermes/src/doc_examples.rs

```diff
@@ -0,0 +1,32 @@
+use crate::store::types::UnixTimestamp;
+
+// Example values for the utoipa API docs.
+// Note that each of these expressions is only evaluated once when the documentation is created,
+// so the examples don't auto-update over time. We may want to adjust these examples for
+// stable/beta in the future so that the example values in the docs work for both hermes and hermes-beta.
+// (Currently all example values are for stable.)
```

Comment:
there's probably a smart way to populate these values on initialization. I'm not sure what the best way to do it is though.

Reply:
[lazh_static](https://docs.rs/lazy_static/latest/lazy_static/)?